tutorials/tts/Audio_Codec_Training.ipynb

```diff
@@ -746,44 +747,44 @@
     },
     {
       "cell_type": "markdown",
-      "source": [
-        "# Related Information"
-      ],
       "metadata": {
         "id": "rynZYwg2VP5d"
-      }
+      },
+      "source": [
+        "# Related Information"
+      ]
     },
     {
       "cell_type": "markdown",
+      "metadata": {
+        "id": "_LtyHHuLkNDv"
+      },
       "source": [
         "To learn more about audio codec models in NeMo, look at our [documentation](https://docs.nvidia.com/nemo-framework/user-guide/latest/nemotoolkit/tts/models.html#codecs).\n",
         "\n",
         "For more information on how to download and run pre-trained audio codec models, visit [NGC](https://catalog.ngc.nvidia.com/models?filters=&orderBy=scoreDESC&query=codec)."
-      ],
-      "metadata": {
-        "id": "_LtyHHuLkNDv"
-      }
+      ]
     },
     {
       "cell_type": "markdown",
-      "source": [
-        "# References"
-      ],
       "metadata": {
         "id": "LeqV3VvJVOb-"
-      }
+      },
+      "source": [
+        "# References"
+      ]
     },
     {
       "cell_type": "markdown",
+      "metadata": {
+        "id": "Rvu4w2x_3RSY"
+      },
       "source": [
         "1.   [EnCodec](https://arxiv.org/abs/2210.13438)\n",
         "2.   [Finite Scalar Quantization (FSQ)](https://arxiv.org/abs/2309.15505)\n",
         "3.   [HiFi-GAN](https://arxiv.org/abs/2010.05646)\n",
         "4.   [SEANet](https://arxiv.org/abs/2009.02095)"
```

Comment:
Should we add our 2 papers to the references?

Reply:
Done